app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,66 @@
+@import "welcome";
+@import "bootstrap-sprockets";
+@import "bootstrap";
+
+/*
+colors for site
+dark blue = #000566
+medium blue = #5FBDF5
+light blue = #C4EAFF
+green = #39B348
+white = #FFF
+*/
```

Comment:
old man memory, I can remove it I just found a color schema I liked and didn't want to forget it.